gcloud-java-storage/src/main/java/com/google/gcloud/storage/testing/FakeStorageRpc.java

```diff
@@ -91,8 +93,49 @@ public StorageObject create(StorageObject object, InputStream content, Map<Optio
   @Override
   public Tuple<String, Iterable<StorageObject>> list(String bucket, Map<Option, ?> options)
       throws StorageException {
-    potentiallyThrow(options);
-    return null;
+    String preprefix = "";
+    for (Map.Entry<Option, ?> e : options.entrySet()) {
+      switch (e.getKey()) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [A switch statement does not contain a break](https://www.codacy.com/app/mziccard/gcloud-java/file/2403961858/issues/source?bid=3123107&fileBranchId=3225960#l98)